lib/src/models/interactive_message/ds_interactive_message_action.model.dart

```diff
@@ -5,12 +5,11 @@ class DSInteractiveMessageAction {
   final String? button;
   final List<DSInteractiveMessageButton>? buttons;
   final List<DSInteractiveMessageSection>? sections;
+  final String? name;
+  final Map<String, dynamic>? parameters;
 
-  DSInteractiveMessageAction({
-    this.button,
-    this.buttons,
-    this.sections,
-  });
+  DSInteractiveMessageAction(
+      {this.button, this.buttons, this.sections, this.name, this.parameters});
```

Comment:
You should add a trailing comma


```suggestion
  DSInteractiveMessageAction({
    this.button,
    this.buttons,
    this.sections,
    this.name,
    this.parameters,
  });
```

Reply:
Done!